src/Services/AdminsService.js

```diff
@@ -0,0 +1,35 @@
+import ApiService from './ApiService';
+
+const { axiosRequest } = ApiService();
+
+const GetAdmins = async (pageNumber, pageSize) => {
```

Comment:
no biggie, but @tackc i think semantically only components and services are `PascalCase`-- i think these methods should be `camelCase`

Reply:
If you're ok with it, I will fix this in a separate PR - I tried to keep my casing consistent with the other Services. We should probably update all of them to be semantically correct

Reply:
no prob! its small potatoes. feel free to leave it